api/insight/apiroutes.go

```diff
@@ -0,0 +1,393 @@
+// Copyright (c) 2017, The dcrdata developers
+// See LICENSE for details.
+
+package insight
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/decred/dcrd/dcrjson"
+	"github.com/decred/dcrd/rpcclient"
+	apitypes "github.com/decred/dcrdata/api/types"
+	"github.com/decred/dcrdata/db/dbtypes"
+	"github.com/decred/dcrdata/db/dcrpg"
+	m "github.com/decred/dcrdata/middleware"
+	"github.com/decred/dcrdata/semver"
+)
+
+type insightApiContext struct {
+	nodeClient *rpcclient.Client
+	BlockData  *dcrpg.ChainDBRPC
+	Status     apitypes.Status
+	statusMtx  sync.RWMutex
+	JSONIndent string
+}
+
+// NewInsightContext Constructor for insightApiContext
+func NewInsightContext(client *rpcclient.Client, blockData *dcrpg.ChainDBRPC, JSONIndent string) *insightApiContext {
+	conns, _ := client.GetConnectionCount()
+	nodeHeight, _ := client.GetBlockCount()
+	version := semver.NewSemver(0, 0, 1)
```

Comment:
please set to whatever api version we're implementing

Reply:
According to Insight docs, they are at API version 1.0

https://documentation.riada.se/insight/latest/insight-advanced-usage-guide/insight-for-developers/insight-rest-api

So 1.0.0 here.